shardy/dialect/sdy/ir/attrs.td

```diff
@@ -74,8 +75,8 @@ def Sdy_Mesh : AttrDef<Sdy_Dialect, "Mesh"> {
       ordering: <["a"=3, "b"=2], device_ids=[0, 2, 4, 1, 3, 5]>
   }];
   let parameters = (ins
-      OptionalArrayRefParameter<"MeshAxisAttr">:$axes,
-      OptionalArrayRefParameter<"int64_t">: $device_ids
+      OptionalArrayRefParameter<"MeshAxisAttr", "Mesh axes">:$axes,
```

Comment:
I guess this should be lower case, why did you go with lower case btw? does it make more sense when displayed in the table?

Reply:
Because we're not writing in full sentences here, capitalization is arbitrary. Since I see most of the previous content was using lowercase, I am guessing this makes it easier to keep consistent in the future.

Reply:
sounds good, thanks!